acd-config-editor/docker_deploy_service.md

```diff
@@ -0,0 +1,48 @@
+# Watson Health Cognitive Services (WHCS) Docker Instructions
+
+## Prerequisites
+* Install Docker
+  * Install the Docker engine from https://docs.docker.com/install/
+  * Verify Docker is installed using the following command
+    ```
+    docker --version
+    ```
+  * If ports are restricted in docker, make sure that port 9272 is opened
+* Install Java
+  * Install the Java 1.8 JRE from https://openjdk.java.net/install/
+  * Verify Java is installed using the following command
+    ```
+    java -version
+    ```
+* Access to Docker images through one of the following:
+  * IBM Cloud CLI - For details see https://cloud.ibm.com/docs/services/Registry?topic=registry-getting-started#getting-started
+    ```
+    ibmcloud login --apikey **\<your-api-key\>** -a https://api.ng.bluemix.net
```

Comment:
Is the install path here still from the IBM Cloud, or does this still need to be updated to use the Azure registry? I would think updates are needed and then the install path tested after we make the doc updates.